src/server-middleware/api/jira/service-desk.js

```diff
@@ -45,10 +52,14 @@ const jiraOptions = options => ({
 });
 
 export default (options = {}) => (req, res) => {
-  // Docs: https://developer.atlassian.com/cloud/jira/service-desk/rest/api-group-request/#api-rest-servicedeskapi-request-post
-  return axios
-    .create({ baseURL: options.origin })
-    .post(JIRA_SERVICE_DESK_API_PATH, jiraData(options, req), jiraOptions(options))
-    .then(jiraRes => res.sendStatus(jiraRes.status))
-    .catch(error => errorHandler(res, error));
+  if (validateReq(req)) {
```

Comment:
Let's make the validation function return a promise, so that we can write something like:
```js
  return validateReq(req)
    .then(() => (axios
      .create({ baseURL: options.origin })
      .post(JIRA_SERVICE_DESK_API_PATH, jiraData(options, req), jiraOptions(options))
      .then(jiraRes => res.sendStatus(jiraRes.status))
    ))
    .catch(() => errorHandler(res, error));
```